contracts/axelar-auth-verifier/src/testutils.rs

```diff
@@ -25,6 +25,16 @@ pub fn randint(a: u32, b: u32) -> u32 {
     rand::thread_rng().gen_range(a..b)
 }
 
+pub fn generate_random_payload_and_hash(env: &Env) -> BytesN<32> {
+    let payload = <soroban_sdk::BytesN<10> as soroban_sdk::testutils::BytesN<10>>::random(env);
```

Comment:
look at existing usage or the docs to simplify code
```suggestion
    let payload = BytesN<10>::random(env);
```

Reply:
Changed to [this](https://github.com/axelarnetwork/axelar-cgp-soroban/pull/8#discussion_r1524310085)